tests/system/providers/amazon/aws/example_athena.py

```diff
@@ -117,6 +119,7 @@ def read_results_from_s3(bucket_name, query_execution_id):
         query=query_read_table,
         database=athena_database,
         output_location=f"s3://{s3_bucket}/",
+        sleep_time=1,
```

Comment:
I dont think we should include this in the documentation. We might want to set it outside of the doc tag. e.g. `read_table.sleep_time = 1`